### DISCUSSION TITLE
HideHudComponentThisFrame ---> HideHudComponent

### DISCUSSION THREAD

**Discussion by BlackBhul** [citizenfx/fivem#1771] [Engineering Discussion]
Hi, what about insert a function like https://docs.fivem.net/natives/?_0x6806C51AD12B83B8 (HideHudComponentThisFrame) but that doesn't require to be inserted in a loop?

**Comment by blattersturm** [comment]
what about it?

**Reply by BlackBhul** [reply]
what?

**Comment by BlackBhul** [comment]
what do u mean
